sample/src/main/kotlin/com/slack/circuit/sample/petlist/PetList.kt

```diff
@@ -185,14 +221,82 @@ private fun PetList(
   animals: List<PetListAnimal>,
   events: (PetListScreen.Event) -> Unit
 ) {
-  LazyColumn(modifier) {
-    animals.forEach { animal ->
-      item { PetListItem(animal) { events(PetListScreen.Event.ClickAnimal(animal.id)) } }
+  LazyVerticalGrid(
+    columns = GridCells.Fixed(2),
+    modifier = modifier,
+    verticalArrangement = Arrangement.spacedBy(16.dp),
+    horizontalArrangement = Arrangement.spacedBy(16.dp),
+    contentPadding = PaddingValues(16.dp),
+  ) {
+    items(
+      count = animals.size,
+      key = { i -> animals[i].id },
+    ) { index ->
+      val animal = animals[index]
+      PetListItem(modifier, animal) { events(PetListScreen.Event.ClickAnimal(animal.id)) }
     }
   }
 }
 
 @Composable
-private fun PetListItem(animal: PetListAnimal, onClick: () -> Unit) {
-  Text(modifier = Modifier.clickable { onClick() }, text = animal.name)
+private fun PetListItem(modifier: Modifier, animal: PetListAnimal, onClick: () -> Unit) {
+  // Palette for extracted colors from the image
+  var paletteState by remember { mutableStateOf<Palette?>(null) }
+  val swatch = paletteState?.getSwatch()
+  val defaultColors = CardDefaults.cardColors()
+  val colors =
+    swatch?.let { s ->
+      CardDefaults.cardColors(
+        containerColor = Color(s.rgb),
+      )
+    }
+      ?: defaultColors
```

Comment:
nit: can you move this up after the `}` (line 252), or move the let to it's own line so this lines up with the `}`? It looks weird and out of place where its at

Reply:
This is what the formatter does, file a feature request with them :)